src/drawers/arcDrawer.ts

```diff
@@ -6,10 +6,28 @@ export module Drawers {
 
     constructor(dataset: Dataset) {
       super(dataset);
-      this._className = "arc";
+      this._className = "arc fill";
       this._svgElementName = "path";
     }
 
+    protected _applyDefaultAttributes(selection: d3.Selection<any>) {
+      super._applyDefaultAttributes(selection);
+      selection.style("stroke", "none");
+    }
+  }
+
+  export class ArcOutline extends Drawer {
```

Comment:
We have to think about the name of this class.  Not sure about `ArcOutline`.  Maybe we should name this thing to be `Arc` and rename the odd `Arc` to be `Sector` maybe?